.gitignore

```diff
@@ -4,5 +4,6 @@ jspm_packages
 
 # Serverless directories
 .serverless
+*-function.json
```

Comment:
I'm not sure if we should commit these files and if so.... is there a way to put them somewhere else other than the root folder?